debugger.md

```diff
@@ -0,0 +1,83 @@
+#Preview: Instructions for setting up the .NET Core debugger
+*Updated on  March 1st 2016 - setup works for Linux & Mac.*
+
+***Currently working within Microsoft Corp-Net only.***
+
+Thank you for taking the time to try out our bits. 
+This page gives you detailed instructions on how to get your system running. 
+It will be updated on a regulary basis to always reflect the latest behaviour. Now let's get started.​
+
+####Your Feedback​
+
+Please place your feedback in this onenote [here](https://microsoft.sharepoint.com/teams/DD_VSPlat/Diagnostics/_layouts/15/WopiFrame.aspx?sourcedoc={04f36c35-0332-4e99-85f7-a33a4c2987e1}&action=edit&wd=target%28%2F%2FDogfood%20Feedback.one%7C4713cc8e-053b-4a0c-9828-745429ee7f45%2F2016Feb%20CoreCLR%20Debugging%20on%20Mac%2C%20Linux%2C%20Windows%7C890db52c-40ef-4181-ae03-d66d9ee0fe54%2F%29). Simply add a new line for every finding. 
+
+####Prerequisites / Known Issues / Notes
+* This is a preview!
+* X64 only
+* Basic Debugging Setup successfully validated on Ubuntu Linux and Mac
+* Currently NOT working on Windows
+* In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. Currently manual steps are required.
+* If you don’t have mono installed you won’t get IntelliSense. See optional installation section below.
+* **Hint:** Setup for Debugging CLR on OSX changed fundamentally end of February 2016. For existing projects please make sure to update your launch.json file according to these steps.​
+
+###First Time setup
+1. Get Visual Studio Code
+    * Install Visual Studio Code. Pick the latest insider version from here: https://code.visualstudio.com/insiders
+    * **Hint:** While we are in a pre-release phase vscode might ask you to install a C# extension from the marketplace. DO NOT INSTALL IT! This extension doesn't contain debugger bits yet and might bring your system in a weird state.   
+2. Install Dotnet CLI
+ * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
+ * **Hint for Linux**:  There's one point where you have to do things different - we need a later version of dotnet: 
```

Comment:
They changed the Linux page last night / this morning so you should no longer need the Linux-specific steps.